index.js

```diff
@@ -30,7 +30,14 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  if( movies === []){
```

Comment:
This condition won't work because _no_ array is equal to another array, no matter their contents. They're different arrays, and therefore different values.

See if you can figure out how to check if an array is empty—it's a test you have to run fairly often!